src/Benchmarks.ClientJob/project.json

```diff
@@ -13,9 +13,12 @@
     "Newtonsoft.Json": "8.0.3"
   },
   "frameworks": {
-    "netstandardapp1.5": {
```

Comment:
Maybe, then they should have been targeting netstandard1.x instead of netstandardapp